lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtTimeConversion.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.routing.pt;
+
+import org.eclipse.mosaic.rti.TIME;
+
+import java.time.LocalTime;
+import java.util.Date;
+
+/**
+ * Contains functions to calculate between different time formats, since "time" is usually always referring to
+ * simulation time, but a real-world time is needed for the calculation of public transportation routes
+ */
+public final class PtTimeConversion {
```

Comment:
Is this something we may want to integrate into the `TIME` class or make it more generic by calling it `TimeUtils` and put it in mosaic objects?

Also I would've thought that we did something like this before.